gramex/apps/nlg/gramex.yaml

```diff
@@ -28,7 +40,7 @@ url:
     pattern: /$YAMLURL/nlg.js
     handler: FileHandler
     kwargs:
-      path: $YAMLPATH/js/nlg.js
+      path: $YAMLPATH/nlg.js
```

Comment:
Don't need to explicit endpoint for directly mapped file. Try it by removing this?

Reply:
@jaidevd -- `nlg.js` will be served by default handler, don't need this definition perhaps?

Reply:
@pratapvardhan This works when running `gramex/apps/nlg` independently, this app is also meant to be imported into other apps. It gives a 404 when running this app while imported in another one.